ui/components/Calendar.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Calendar, momentLocalizer } from "react-big-calendar";
+import moment from "moment";
+import PropTypes from "prop-types";
+import "react-big-calendar/lib/css/react-big-calendar.css";
+
+const localizer = momentLocalizer(moment);
+
+PerformanceCalendar.propTypes = {
+  title: PropTypes.string.isRequired,
+  events: PropTypes.array.isRequired,
+  defaultDate: PropTypes.instanceOf(Date),
+  defaultView: PropTypes.string,
+};
+
+PerformanceCalendar.defaultProps = {
+  defaultDate: moment().toDate(),
+  defaultView: "month",
+};
+
+export const PerformanceCalendar = ({ title, events, defaultDate, defaultView }) => {

```

Comment:
This is Ok. But we also need another calendar component which has to be generic and this `PerformanceCalendar` component has to be created from that generic component.

Reply:
I see, I think we can change the name of this generic Calendar component from `PerformanceCalendar` to `MesheryCalendar`.  And then we can re-use this component when creating an actual Performance Calendar on the Performance page.

Reply:
Yep